6_Agent_Deployment/backend_agent_api/agent_api.py

```
@@ -493,6 +493,133 @@ async def reset_failed_processing():
+class FileUploadRequest(BaseModel):
+    """Request model for processing uploaded files."""
+    document_id: str
+    file_path: str
+    file_name: str
+    mime_type: str
+
+
+@app.post("/api/process-upload")
+async def process_upload(
+    request: FileUploadRequest,
+    user: Dict[str, Any] = Depends(verify_token),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)